### DISCUSSION TITLE
Tab for the preview window

### DISCUSSION THREAD

**Discussion by hadrien-iconosquare** [Laravel-Backpack/community-forum#293] [Ideas]
Hello everybody
I would like to implement a tab system for the preview window like the add and edit windows. I think implementing it would be crazy. Plus I saw that someone already made a merge request about it: [https://github.com/Laravel-Backpack/CRUD/pull/3314](https://github.com/Laravel-Backpack/CRUD/pull/3314)

Would it be possible to give me a trick to do this without changing the vendor folder while waiting for a new feature. I thank you in advance for your help.

Thank you in advance for your help.

**Comment by pxpm** [comment]
This fell through the cracks. Sorry @hadrien-iconosquare 

It's already been done, and merged here: https://github.com/Laravel-Backpack/CRUD/pull/5043

Cheers
